projects/angular/src/data/tree-view/_properties.tree-view.scss

```diff
@@ -4,38 +4,43 @@
-      // Usage: ../data/tree-view/_tree-view.clarity.scss
-      --clr-tree-border-radius: var(--clr-global-borderradius);
-
-      // --clr-custom-links-hover-color is deprecated in v17 to be deleted in v18.
-      // Remove reference during shim removal.
-      --clr-tree-link-hover-color: var(--clr-custom-links-hover-color);
-      --clr-tree-link-selection-color: var(--clr-global-selection-color);
-      --clr-tree-link-text-color: var(--clr-color-neutral-700);
-
-      --clr-tree-caret-size: #{tree-view-variables.$clr-tree-caret-size};
-      --clr-tree-node-touch-target: #{tree-view-variables.$clr-tree-node-touch-target};
-      --clr-tree-caret-padding: #{tree-view-variables.$clr-tree-caret-padding};
-      --clr-tree-node-content-icon-margin-right: #{tree-view-variables.$clr-tree-node-content-icon-margin-right};
-      --clr-tree-node-caret-link-hover-color: var(--clr-color-neutral-1000);
-      --clr-tree-node-caret-color: var(--clr-color-neutral-500);
-      --clr-tree-node-caret-link-active-color: var(--clr-color-neutral-500);
-      --clr-tree-node-caret-link-disabled-color: var(--clr-color-neutral-500);
-
-      --clr-tree-node-font-size: #{tree-view-variables.$clr-tree-node-font-size};
-      --clr-tree-node-font-weight: #{tree-view-variables.$clr-tree-node-font-weight};
-      --clr-tree-node-line-height: #{tree-view-variables.$clr-tree-node-line-height};
-      --clr-tree-node-letter-spacing: #{tree-view-variables.$clr-tree-node-letter-spacing};
-      --clr-tree-node-content-padding: #{0 0 0 tree-view-variables.$clr-tree-node-link-padding-left};
-      --clr-tree-node-content-margin-left: #{tree-view-variables.$clr-tree-node-touch-target};
-
-      --clr-tree-node-children-margin: #{tree-view-variables.$clr-tree-node-children-margin};
-    }
+  :root [cds-theme] {
+    // Usage: ../data/tree-view/_tree-view.clarity.scss
+    --clr-tree-border-radius: #{tokens.$cds-alias-object-border-radius-100};
+
+    // @deprecated in v17 replaced by clr-tree-node-caret-link-color. Will be removed in v18.
+    --clr-tree-node-caret-color: #{tokens.$cds-alias-object-interaction-color};
+    // @deprecated in v17 in favor of clr-tree-node-content-bg-hover-color
+    --clr-tree-link-hover-color: #{tokens.$cds-alias-object-interaction-background-hover};
+    // @deprecated in v17 in favor of clr-tree-node-content-bg-selected-color
+    --clr-tree-link-selection-color: #{tokens.$cds-alias-object-interaction-background-selected};
+    // @deprecated in v17 in favor of clr-tree-node-content-color
+    --clr-tree-link-text-color: #{tokens.$cds-alias-object-interaction-color};
+
+    --clr-tree-node-caret-link-color: var(--clr-tree-node-caret-color);
+    --clr-tree-node-caret-link-hover-color: #{tokens.$cds-alias-object-interaction-color-hover};
+    --clr-tree-node-caret-link-active-color: #{tokens.$cds-alias-object-interaction-color-active};
+    --clr-tree-node-caret-link-disabled-color: #{tokens.$cds-alias-object-interaction-color-disabled};
```

Comment:
Do we really need these to be named like that? The class which is using these is `clr-treenode-link` so the name `caret-link-color` sound like setting the focus on the caret while it should be the link